src/redist/targets/GenerateLayout.targets

```diff
@@ -274,6 +267,7 @@
         <BaseUrl Condition=" '$(DotNetBuildFromSource)' == 'true' AND '$(DotNetBuildFromSourceFlavor)' == 'Product' ">$(AspNetCoreSharedFxRootUrl)</BaseUrl>
         <DownloadFileName>$(AspNetCoreSharedFxArchiveFileName)</DownloadFileName>
         <RelativeLayoutPath></RelativeLayoutPath>
+        <DirectoriesToCopy>shared/Microsoft.AspNetCore.App</DirectoriesToCopy>
```

Comment:
We need to make this same change on the WindowsDesktop shared framework, as that one can cause the same problem on Windows.